### PR TITLE
[ADD] delivery_carrier_label_ups: Support customer returns

### PR DESCRIPTION
**Pull request by hbrunn**
this allows to create pickup requests and makes the label attached to the picking an [electronic return label](https://www.ups.com/us/en/help-center/sri/glo-erl.page)

TODO

- [x] provide button so that you can request an ERL manually at any time for UPS returns

### CODE REVIEW

File: delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py [hunk 1]
```diff
@@ -99,3 +99,15 @@ def test_labels(self):
                 ),
             )
             return super().test_labels()
+
```
Comment (StefanRijnhart) [inline delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py:106]: `delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py:104:5: E303 too many blank lines (2)`

Comment (hbrunn) [reply delivery_carrier_label_ups/tests/test_delivery_carrier_label_ups.py:106]: thanks, I'll have to rework some things here anyways, as I realized yesterday that it's nonsense to handle the pickup when the picking is done. This needs to happen earlier, and I'm not yet sure if I'll add some custom stuff here or extend base...label to allow a general mechanism for pickups.

**Comment by hbrunn** [comment]
this needs to be adopted to https://github.com/OCA/delivery-carrier/pull/352 and only merged afterwards

File: delivery_carrier_label_ups/models/stock_picking.py [hunk 2]
```diff
@@ -13,6 +13,19 @@
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
+    @api.depends(
+        'delivery_type', 'location_id.usage', 'carrier_tracking_ref', 'state'
+    )
+    def _compute_show_label_button(self):
```
Comment (StefanRijnhart) [inline delivery_carrier_label_ups/models/stock_picking.py:19]: A docstring mentioning `returns` will be appreciated I think.

Comment (hbrunn) [reply delivery_carrier_label_ups/models/stock_picking.py:19]: done

**Review by StefanRijnhart — APPROVED** [review]

**Review by astirpe — APPROVED** [review]
Code review only (not tested).
There is one conflict to be resolved.

**Comment by hbrunn** [comment]
thanks for the heads up, rebased

**Pull request merged**